crates/core/src/db/relational_db.rs

```diff
@@ -139,12 +140,29 @@ impl RelationalDB {
             commit_log,
             _lock: Arc::new(lock),
             address: db_address,
+            row_count_fn: Arc::new(move |table| {
+                METRICS
+                    .rdb_num_table_rows
+                    .with_label_values(&db_address, &table.into())
+                    .get()
+            }),
         };
 
         log::info!("[{}] DATABASE: OPENED", address);
         Ok(db)
     }
 
+    /// Returns an approximate row count for a particular table.
```

Comment:
There is already a `RowCount` object:

```rust
#[derive(Debug, Copy, Clone, PartialOrd, Ord, PartialEq, Eq)]
pub struct RowCount {
    pub min: usize,
    pub max: Option<usize>,
}
```

on `sats -> Relation`

Reply:
I will add a TODO like you mentioned.

I wonder if we should move `Relation` out of `sats`, since it will need access to database statistics. Because right now `RowCount` only works on `MemTable`s.